scripts/rpm-prepare.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set +ex
+
+echo "Reading ODO_VERSION, ODO_RELEASE and GIT_COMMIT env, if they are set"
+export ODO_VERSION=${ODO_VERSION:=1.0.0-test}
+export ODO_RELEASE=${ODO_RELEASE:=1}
+export GIT_COMMIT=${GIT_COMMIT:=`git rev-parse --short HEAD 2>/dev/null`}
+export ODO_RPM_VERSION=${ODO_VERSION//-}
+
+# Print env for verifcation
+echo "Printing envs for verification"
+echo "ODO_VERSION=$ODO_VERSION"
+echo "ODO_RELEASE=$ODO_RELEASE"
+echo "GIT_COMMIT=$GIT_COMMIT"
+echo "ODO_RPM_VERSION=$ODO_RPM_VERSION"
+
+OUT_DIR=".rpmbuild"
+DIST_DIR="$(pwd)/dist"
+
+SPEC_DIR="$OUT_DIR/SPECS"
+SOURCES_DIR="$OUT_DIR/SOURCES"
+FINAL_OUT_DIR="$DIST_DIR/rpmbuild"
+
+NAME="atomic-openshift-odo-$ODO_RPM_VERSION-$ODO_RELEASE"
+
+echo "Making release for $NAME, git commit $GIT_COMMIT"
+
+echo "Cleaning up old content"
+if [[ -d $DIST_DIR ]]; then
```

Comment:
Since we're doing an `rm -rf`, why do we even need the `if` checks in these two cases?

Reply:
hmm makes sense as rm -rf never fails but still., it's better to check. file exists before any operation.